templates/js/controllers/userController.default.js

```diff
@@ -1,3 +1,3 @@
-export const getAllUsers = (req, res, next) => {
+export const getAllUsers = (req, res) => {
```

Comment:
I think it is a good idea to keep the `next` middleware. It is helpful in case the current middleware doesn't return a response. Reference: https://expressjs.com/en/guide/using-middleware.html

Reply:
i agree that `next` should be used, the linter complains about unused variable/param
This "default" controller will only be used if the user selects "none" as a database option

Reply:
Could we add next part to be ignored in .eslintignore?

Reply:
Found this:

```
"eslintConfig": {
    "rules": {
      "no-unused-vars": ["error", { "argsIgnorePattern": "next" }]
    }
  }
```

Reply:
oooo nice find, let me add this